cdap-ui/app/cdap/components/Footer/index.js

```diff
@@ -15,10 +15,15 @@
  */
 
 import React, { PureComponent } from 'react';
+import {objectQuery} from '../../services/helpers';
```

Comment:
just `services/helpers` don't work?

Reply:
Not sure why, but when I imported this without ellipses, I got a `Can't resolve 'services/helpers'`, when importing `Footer` component in `app/login/login.js`.

Reply:
that's probably because the alias is not defined in the login webpack file